sound/soc/sof/topology.c

```diff
@@ -1858,7 +1862,7 @@ static int sof_link_load(struct snd_soc_component *scomp, int index, struct snd_
 			return ret;
 	}
 
-	/* Set nonatomic property for FE dai links as their trigger action involves IPC's */
+	/* Set nonatomic property for FE dai links for FE-BE compatibility */
```

Comment:
did you update the comment in the wrong place?

Reply:
Yes, I did :(